lib/msal-core/src/utils/CryptoUtils.ts

```diff
@@ -83,6 +83,15 @@ export class CryptoUtils {
         }
     }
 
+    /**
+     * verifies if a string is  GUID
+     * @param guid
+     */
+    static isGuid(guid: string) {
```

Comment:
Are there tests for this?

Reply:
Yes. In ServerRequestParameters.spec.ts indirectly. I can add one for cryptoUtils.

Reply:
Added tests for CryptoUtils.ts (I added all tests, not just for this API since it is not much of an effort)